src/methods/get.js

```diff
@@ -549,3 +552,23 @@ export function GetTemplateObject(realm: Realm, templateLiteral: BabelNodeTempla
   // 15. Return template.
   return template;
 }
+
+export function GetFromArrayWithWidenedNumericProperty(realm: Realm, arr: ArrayValue, P: string | SymbolValue): Value {
+  let type = Value;
+
+  if (typeof P === "string") {
+    // these are safe methods to allow, as they return a new array
+    // so we use the ordinary get for these cases. Reduce can be
+    // unsafe, but we check for that in the prototype method
```

Comment:
Reduce is not guaranteed to return an array, and we removed that logic. Let's fix up the comment too.